src/main/java/net/mcreator/ui/MCreatorApplication.java

```diff
@@ -142,6 +143,10 @@ private MCreatorApplication(List<String> launchArguments) {
 		// load entity animations for the Java Model animation editor
 		EntityAnimationsLoader.init();
 
+		// register mod element types
+		// TODO: Move mod elements to the plugin system

```

Comment:
Master code should not have todos, I suggest you run own todo board for todo notes